### PR TITLE
Tags: improve logging and operations' verification

### PR DESCRIPTION
**Pull request by drew2a**
This PR lowers validation error level to `warning` after the report from  @ichorid about "tags error verification" in logs:

```python
[PID:37904] 2021-12-09 17:32:31,508 - ERROR <tag_community:82> TagCommunity.on_message(): Invalid signature for b'\xf0,\xfd\x82\x13\xbcw\x1fA\xc1\xec\xdcA\xd9\xf5\xad\x8f\x82,V\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00\x00\x00\x00\x01LibNaCLPK:\x95\xf81\xe5\\\xceug\xa6\xed6\xd2\x06Y\xfe\xb4\xd2\xe3\x94\x9e6A5D\xad}\xab\x1ca\xb6)G-\xff\xe5\xeb\xc3\xcf\xf0H\tXvA\x83\x9f\xfa\xab\xad\xb0A}\x0e\xe6\xfaCJ\xd4\xe3\tE\xa7\x0c\xaa\x00\x04720p'
[PID:37904] 2021-12-09 17:32:31,509 - ERROR <tag_community:82> TagCommunity.on_message(): Invalid signature for b'\xf0,\xfd\x82\x13\xbcw\x1fA\xc1\xec\xdcA\xd9\xf5\xad\x8f\x82,V\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00\x00\x00\x00\x01LibNaCLPK:\x95\xf81\xe5\\\xceug\xa6\xed6\xd2\x06Y\xfe\xb4\xd2\xe3\x94\x9e6A5D\xad}\xab\x1ca\xb6)G-\xff\xe5\xeb\xc3\xcf\xf0H\tXvA\x83\x9f\xfa\xab\xad\xb0A}\x0e\xe6\xfaCJ\xd4\xe3\tE\xa7\x0c\xaa\x00\x0510bit'
[PID:37904] 2021-12-09 17:32:31,510 - INFO - TagCommunity(75) - Tag added: video:b'\xd4A$\x95\x86\xc4|&N\xe4M.\xb7\xda\xdd\xa7\x19Z3v'
[PID:37904] 2021-12-09 17:32:31,511 - ERROR <tag_community:82> TagCommunity.on_message(): Invalid signature for b'\x7f\xf1\xbb\xb6\xb4\xe3\x02+\xea\xb5\xdd\xdb\x1fIsO\x9b\x15zk\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00\x00\x00\x00\x01LibNaCLPK:\x95\xf81\xe5\\\xceug\xa6\xed6\xd2\x06Y\xfe\xb4\xd2\xe3\x94\x9e6A5D\xad}\xab\x1ca\xb6)G-\xff\xe5\xeb\xc3\xcf\xf0H\tXvA\x83\x9f\xfa\xab\xad\xb0A}\x0e\xe6\xfaCJ\xd4\xe3\tE\xa7\x0c\xaa\x00\x0510bit'
[PID:37904] 2021-12-09 17:32:31,511 - ERROR <tag_community:82> TagCommunity.on_message(): Invalid signature for b"C\xcc\xa6f\x07\x8b6\xe2\xdd\xf6uMh\xa1\x05\xc9O%\x1a'\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00\x00\x00\x00\x01LibNaCLPK:\x95\xf81\xe5\\\xceug\xa6\xed6\xd2\x06Y\xfe\xb4\xd2\xe3\x94\x9e6A5D\xad}\xab\x1ca\xb6)G-\xff\xe5\xeb\xc3\xcf\xf0H\tXvA\x83\x9f\xfa\xab\xad\xb0A}\x0e\xe6\xfaCJ\xd4\xe3\tE\xa7\x0c\xaa\x00\x06webrip"
[PID:37904] 2021-12-09 17:32:31,511 - ERROR <tag_community:82> TagCommunity.on_message(): Invalid signature for b'\x7f\xf1\xbb\xb6\xb4\xe3\x02+\xea\xb5\xdd\xdb\x1fIsO\x9b\x15zk\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00\x00\x00\x00\x01LibNaCLPK:\x95\xf81\xe5\\\xceug\xa6\xed6\xd2\x06Y\xfe\xb4\xd2\xe3\x94\x9e6A5D\xad}\xab\x1ca\xb6)G-\xff\xe5\xeb\xc3\xcf\xf0H\tXvA\x83\x9f\xfa\xab\xad\xb0A}\x0e\xe6\xfaCJ\xd4\xe3\tE\xa7\x0c\xaa\x00\x04x265'
[PID:37904] 2021-12-09 17:32:31,512 - ERROR <tag_community:82> TagCommunity.on_message(): Invalid signature for b'\xf0,\xfd\x82\x13\xbcw\x1fA\xc1\xec\xdcA\xd9\xf5\xad\x8f\x82,V\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00\x00\x00\x00\x01LibNaCLPK:\x95\xf81\xe5\\\xceug\xa6\xed6\xd2\x06Y\xfe\xb4\xd2\xe3\x94\x9e6A5D\xad}\xab\x1ca\xb6)G-\xff\xe5\xeb\xc3\xcf\xf0H\tXvA\x83\x9f\xfa\xab\xad\xb0A}\x0e\xe6\xfaCJ\xd4\xe3\tE\xa7\x0c\xaa\x00\x04hevc'
[PID:37904] 2021-12-09 17:32:31,512 - ERROR <tag_community:82> TagCommunity.on_message(): Invalid signature for b'\x7f\xf1\xbb\xb6\xb4\xe3\x02+\xea\xb5\xdd\xdb\x1fIsO\x9b\x15zk\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00\x00\x00\x00\x01LibNaCLPK:\x95\xf81\xe5\\\xceug\xa6\xed6\xd2\x06Y\xfe\xb4\xd2\xe3\x94\x9e6A5D\xad}\xab\x1ca\xb6)G-\xff\xe5\xeb\xc3\xcf\xf0H\tXvA\x83\x9f\xfa\xab\xad\xb0A}\x0e\xe6\xfaCJ\xd4\xe3\tE\xa7\x0c\xaa\x00\x051080p'
[PID:37904] 2021-12-09 17:32:31,513 - ERROR <tag_community:82> TagCommunity.on_message(): Invalid signature for b"C\xcc\xa6f\x07\x8b6\xe2\xdd\xf6uMh\xa1\x05\xc9O%\x1a'\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00\x00\x00\x00\x01LibNaCLPK:\x95\xf81\xe5\\\xceug\xa6\xed6\xd2\x06Y\xfe\xb4\xd2\xe3\x94\x9e6A5D\xad}\xab\x1ca\xb6)G-\xff\xe5\xeb\xc3\xcf\xf0H\tXvA\x83\x9f\xfa\xab\xad\xb0A}\x0e\xe6\xfaCJ\xd4\xe3\tE\xa7\x0c\xaa\x00\x04hevc"
[PID:37904] 2021-12-09 17:32:31,513 - ERROR <tag_community:82> TagCommunity.on_message(): Invalid signature for b"C\xcc\xa6f\x07\x8b6\xe2\xdd\xf6uMh\xa1\x05\xc9O%\x1a'\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00\x00\x00\x00\x01LibNaCLPK:\x95\xf81\xe5\\\xceug\xa6\xed6\xd2\x06Y\xfe\xb4\xd2\xe3\x94\x9e6A5D\xad}\xab\x1ca\xb6)G-\xff\xe5\xeb\xc3\xcf\xf0H\tXvA\x83\x9f\xfa\xab\xad\xb0A}\x0e\xe6\xfaCJ\xd4\xe3\tE\xa7\x0c\xaa\x00\x05video"
```

Also, this PR changes verification logic by replacing `AssertionError` with `ValueError` as @kozlovsky suggested.

PR contains a bug fix for the signing procedure.

### CODE REVIEW

File: src/tribler-core/tribler_core/components/tag/community/tag_community.py [hunk 1]
```diff
@@ -55,36 +57,38 @@ def request_tags(self):
 
         peer = random.choice(self.get_peers())
         self.requests.register_peer(peer, REQUESTED_TAGS_COUNT)
-        self.logger.debug(f'Request {REQUESTED_TAGS_COUNT} tags')
+        self.logger.info(f'-> request {REQUESTED_TAGS_COUNT} tags from peer {hexlify(peer.mid)}')
```
Comment (kozlovsky) [inline src/tribler-core/tribler_core/components/tag/community/tag_community.py:60]: nit: `hexlify` is not necessary anymore, we can use `hex` method of `bytes` object instead:
https://docs.python.org/3/library/stdtypes.html#bytes.hex

```suggestion
        self.logger.info(f'-> request {REQUESTED_TAGS_COUNT} tags from peer {peer.mid.hex()}')
```
(this comment is also applied to all other places where `hexlify` used in this PR)

File: src/tribler-core/tribler_core/components/tag/db/tag_db.py [hunk 2]
```diff
@@ -124,11 +124,11 @@ def add_tag_operation(self, operation: TagOperation, signature: bytes, is_local_
             self.instance.TorrentTagOp(torrent_tag=torrent_tag, peer=peer, operation=operation.operation,
                                        clock=operation.clock, signature=signature)
             torrent_tag.update_counter(operation.operation, is_local_peer=is_local_peer)
-            return
+            return True
```
Comment (kozlovsky) [inline src/tribler-core/tribler_core/components/tag/db/tag_db.py:127]: **issue:** "Returns:" section presents in the docstring, but return value is not described

**Review by kozlovsky — CHANGES_REQUESTED** [review]

File: src/tribler-core/tribler_core/components/tag/community/tag_community.py (hunk 1 above)
Comment (drew2a) [reply src/tribler-core/tribler_core/components/tag/community/tag_community.py:60]: Fixed

File: src/tribler-core/tribler_core/components/tag/db/tag_db.py (hunk 2 above)
Comment (drew2a) [reply src/tribler-core/tribler_core/components/tag/db/tag_db.py:127]: Fixed

**Review by kozlovsky — APPROVED** [review]

**Pull request merged**